backend/functions/src/email/functions.ts

```diff
@@ -0,0 +1,138 @@
+import * as emails from './templates'
+import * as nodemailer from 'nodemailer'
+import { config } from 'dotenv'
+config()
+
+const transport = {
+  host: 'smtp.gmail.com',
+  port: 465,
+  secure: true,
+  auth: {
+    user: process.env.EMAIL_EMAIL,
+    pass: process.env.EMAIL_PASS,
+  },
+}
+
+const transporter = nodemailer.createTransport(transport)
+
+transporter.verify((err, succ) => {
+  if (err) {
+    console.log(' ❌ Transporter error: ❌ \n ', err)
+  } else {
+    console.log(
+      ' \n  ✅ Nodemailer transporter working ✅  \n Successful Message: ',
+      succ
+    )
+  }
+})
```

Comment:
I think you can get rid of these console.log statements, unless they serve a purpose I don't realize.